lib/rules/validate-quote-marks.js

```diff
@@ -106,10 +106,16 @@ module.exports.prototype = {
                     return;
                 }
 
+                token.value = token.value.replace(
+                    new RegExp('^' + mark + '(.*)' + mark + '$'),
```

Comment:
Yeah, came from #1202 that I pulled into this one. It seems to work and pass the tests, so I'm not super worried about it. It might need to change if we try to fix multi line strings, but I wouldn't bother changing it until that comes up. We shouldn't be scared of regex just because it is regex. Even though we both seemed to have the same reaction :P